test/e2e-common.sh

```diff
@@ -314,6 +314,11 @@ function install_knative_serving_standard() {
 	    -f "${SERVING_HPA_YAML}" || return 1
     UNINSTALL_LIST+=( "${SERVING_CORE_YAML}" "${SERVING_HPA_YAML}" )
 
+    echo "Knative TLS YAML: ${SERVING_CERT_MANAGER_YAML} and ${SERVING_NSCERT_YAML}"
```

Comment:
Where are SERVING_CERT_MANAGER_YAML and SERVING_NSCERT_YAML defined?

Reply:
They are defined at https://github.com/knative/serving/blob/b6f188ad89fe540d3d8fb121cea366d9090395ca/hack/generate-yamls.sh#L55.
These two environment variables were set when calling `build_knative_from_source` https://github.com/knative/serving/blob/0be83350bf00a1cb87901b52e05f27e70d623b25/test/e2e-common.sh#L279

I added the comment in the code to make it more clear.